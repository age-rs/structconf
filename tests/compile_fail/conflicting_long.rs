//! Conflicting options for the long argument.

use structconf::StructConf;

#[derive(StructConf)]
struct Config {
    #[conf(no_long, long = "thing")]
    pub value: bool,
}

fn main() {}
