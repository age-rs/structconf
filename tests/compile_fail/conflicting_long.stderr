error: `no_long` is incompatible with `long`.
 --> $DIR/conflicting_long.rs:8:9
  |
8 |     pub value: bool,
  |         ^^^^^
